python_modules/dagster/dagster/_core/definitions/decorators/repository_decorator.py

```diff
@@ -212,96 +212,95 @@ def repository(
         description (Optional[str]): A string description of the repository.
 
     Example:
+        .. code-block:: python
```

Comment:
Did ruff do this reformatting or is this you?

Reply:
So I actually can't remember for that specific one-- at the time of these changes (there have prob been ~20 ruff releases since) ruff was flagging this and not autofixing.

I just experimented on the newest version and it accepts these:

```
Example:
.. code-block::

Example:
    .. code-block::

Example:

    .. code-block::
```

But not this:

```
Example:

.. code-block::
```

Which it autocorrects to:

```
Example:
.. code-block::
```

So probably we should go with that last form, since it has less indentation.